### DISCUSSION TITLE
files doesn't open on a new tab

### DISCUSSION THREAD

**Discussion by volkan0x** [zed-industries/zed#14884] [Feature Requests]
when i try to open a new file, it opens on top of the previous file 

like this:

https://github.com/user-attachments/assets/b313a9e0-a7eb-4e57-a12e-f9e33afff3de

i want to open new files with a new tab is there any shortcut or smt? anyone knows how to do that?

**Comment by SomeoneToIgnore** [comment]
```jsonc
// Settings related to preview tabs.
"preview_tabs": {
  // Whether preview tabs should be enabled.
  // Preview tabs allow you to open files in preview mode, where they close automatically
  // when you switch to another file unless you explicitly pin them.
  // This is useful for quickly viewing files without cluttering your workspace.
  "enabled": true,
  // Whether to open tabs in preview mode when selected from the file finder.
  "enable_preview_from_file_finder": false,
  // Whether a preview tab gets replaced when code navigation is used to navigate away from the tab.
  "enable_preview_from_code_navigation": false
},
```

You're interested in this part of the default settings.

**Reply by volkan0x** [reply]
what should i change here to what setting? and after changing it redirects me to save the file in somewhere in my computer, where should i save the new file?

**Reply by SomeoneToIgnore** [reply]
Please have at least a glance at the documentation first: https://zed.dev/docs/configuring-zed

**Reply by volkan0x** [reply]
omg this documentation saved my life thanks for it.... ⚡️

**Comment by nise** [comment]
This is such a wired default that new user frustrates extremely. Imagine, after installation I just tried to open two text files in separate tabs. It is impossible without reading the manual and understanding the setting.json, seriously?

**Reply by nise** [reply]
The suggestions above do not even solve the problem.

**Comment by nise** [comment]
Ha, it was not even possible to create a new file. I wish good luck.

**Comment by AgilentGCMS** [comment]
OK, I'm done with Zed. I tried it because it had good reviews. But even with
```
"preview_tabs": {
    "enable_preview_file_from_code_navigation": false,
    "enable_preview_from_multibuffer": false,
    "enable_preview_from_project_panel": false,
    "enabled": false
  },
```
I cannot for the life of me open two files at the same time in two tabs. What a frustrating default!

**Reply by volkan0x** [reply]
@AgilentGCMS 

Try double-clicking the open tab; when the tab title changes from italic to normal font, you can now open two files simultaneously in two separate tabs.
